client/state/audio/test/middleware.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy, stub } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import middleware from '../middleware';
+import { HAPPYCHAT_RECEIVE_EVENT } from 'state/action-types';
+
+describe( 'Audio Middleware', () => {
+	let next;
+	let store;
+	let play;
+
+	beforeEach( () => {
+		next = spy();
+
+		store = {
+			dispatch: spy(),
+			getState: stub(),
+			replaceReducers: spy(),
+			subscribe: spy(),
```

Comment:
I won't put my foot down against this but do we really need to mock the full store since these functions don't use anything other than `dispatch`?

we can test to make sure that nobody introduces code changes which do this though I would hope that our review process will catch any of those things. they would be odd anti-patterns anyway if someone used the audio middleware to replace the reducers though maybe not technical bugs